### ISSUE TITLE
MultiSymbol Read for Tickstore

### ISSUE THREAD

**Issue by bmoscon** [manahl/arctic#413]
See #394

**Comment by richardbounds** [comment]
Tickstore already supports this, afaik.

**Comment by bmoscon** [comment]
@richardbounds thanks, I meant VersionStore :)

Will edit title

**Comment by johnjihong** [comment]
any progress on the feature

**Comment by bmoscon** [comment]
not really. Its a LOT of code change due to how version store reads symbols. Its not a simple lookup, it does a lot of work related to the versioning before it actually reads the data. I'll get to it eventually, but there are other more pressing bugs I'm working on at the moment.

**Comment by johnjihong** [comment]
Well noted.
